app/replica.go

```diff
@@ -182,8 +182,9 @@ func startReplica(c *cli.Context) error {
 		server := rest.NewServer(s)
 		router := http.Handler(rest.NewRouter(server))
 		router = util.FilteredLoggingHandler(map[string]struct{}{
-			"/ping":          {},
-			"/v1/replicas/1": {},
+			"/ping":                   {},
+			"/v1/replicas/1":          {},
+			"/v1/replicas/1/volusage": {},
```

Comment:
why are we adding to filteredPaths?

Reply:
So that logs from those paths can't be collected.